projects/components/src/table/table.component.ts

```diff
@@ -293,6 +293,10 @@ export class TableComponent
   }
 
   public ngOnChanges(changes: TypedSimpleChanges<this>): void {
+    if (changes.display) {
+      this.isTableFullPage = this.display === TableStyle.FullPage;
```

Comment:
This should replace line 285 - this.display will never be set in the constructor (inputs come in later)